src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function getSum(firstValue) {
```

Comment:
Not really sure about whether you should name this function or not, but it does improve code readability, good job 👍